### PR TITLE
controller: change event as const

### PR DESCRIPTION
**Pull request by xiang90**
/cc @hongchaodeng @hasbro17

### CODE REVIEW

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -46,6 +46,15 @@ var (
 	initRetryWaitTime = 30 * time.Second
 )
 
+const (
+	// Add event type from Kubernetes watch response
+	watchEventAdd = "ADDED"
```
Comment (hongchaodeng) [inline pkg/controller/controller.go]: This is already defined in client-go watch/
```
const (
	Added    EventType = "ADDED"
	Modified EventType = "MODIFIED"
	Deleted  EventType = "DELETED"
	Error    EventType = "ERROR"

	DefaultChanSize int32 = 100
)
```

Comment (xiang90) [inline pkg/controller/controller.go]: ok. i will wait your client 2.0 pr get merged then.

**Comment by xiang90** [comment]
@hongchaodeng PTAL

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**